### ISSUE TITLE
`morpheval.limsi.v2.en.sents` use `macintosh` encoding

### ISSUE THREAD

**Issue by gsarti** [franckbrl/morpheval_v2#2]
If someone plans to load sentences in Python or similar, be aware that the file is encoded using the `macintosh` charset. This worked for me to get the original sentences in UTF-8:

`iconv -f macintosh -t utf-8 morpheval.limsi.v2.en.sents -o morpheval.limsi.v2.en.sents.utf8`

**Comment by franckbrl** [comment]
Did you get the file from the link specified in the [readme](https://github.com/franckbrl/morpheval_v2#requirements)? Here is what I get right after a wget:
```
$ file -bi morpheval.limsi.v2.en.sents 
text/plain; charset=utf-8
```

**Comment by gsarti** [comment]
Yes, the file itself is encoded in utf-8 but inspecting its contents shows the presence of many error codes derived from a bad format conversion (e.g. look for â€, â€œ, Â· ). These can be fixed by running the aforementioned command, to obtain a "real" utf-8 file. Would it be possible to replace the file linked in the README with one formatted with iconv, so that this step is not needed in the future?
